### PR TITLE
Add VS2015 support

### PR DESCRIPTION
**Pull request by GDI123**
Tested with VS2015 on Windows 10 without internet access.

### CODE REVIEW

**Review by jp31415926 — APPROVED** [review]
After turning on "ignore whitespace changes" it looks good to me.

File: Readme.md [hunk 1]
```diff
@@ -1,73 +1,90 @@
...
+
+### Changelog
+
+- 0.0.1 - Initial commit, VS2017 support
+- 0.0.2 - Added VS2019 support
+- 0.0.3 - Fixed manifest to avoid execution errors under fresh PowerShell environments ([@1Dimitri](https://github.com/1Dimitri))
+- 0.0.4 - Support downlevel PowerShell versions, starting from `3.0`
+- 0.0.5 - Duh, actually set `PowerShellVersion = '3.0'` in manifest
+- 0.0.6 - Load `System.Security` assembly if module was imported without manifest
+- 0.0.7 - Added VS2015 support
```
Comment (beatcracker) [inline Readme.md]: I'd suggest that you put your name here ;). Something like:
```none
- 0.0.7 - Added VS2015 support ([@GDI123](https://github.com/GDI123))
```

**Comment by beatcracker** [comment]
Thanks for the contribution! The code looks good to me, but I'd really like to fix EOL issues.

Looks like part of it is my fault: I think that `* text=auto` converts files to `LF` on commit, but `*.ps1    text  eol=crlf` makes them `CRLF` on checkout. And that's not what I intended and makes the issue hard to spot.

Here's what worked for me, you you woud like to fix it yourself:

1. Clone https://github.com/GDI123/VSCELicense in a separate folder
2. Create `attributes` file in `.git\info` subfolder, containing:
```none
* -text

```
3. Delete all files in the repo except `.git` folder.
4. Do `git checkout master --force`

After this you'll see that files modified by you have `LF`  for newlines and then you can convert them to `CRLF` and update PR. This should get red of the whitespace issue.

Alternatively, since you've enabled [maintainers to make changes](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests), I can try to fix it myself.

**Comment by GDI123** [comment]
Your advise doesn't worked for me. I did all options (add file, delete all except .git), but got the message:
```
$ git checkout master --force
Already on 'master'
Your branch is up to date with 'origin/master'.
```
And also in my local repo I'm already have `.gitattributes` file which contains:
```
# Set default behaviour, in case users don't have core.autocrlf set.
* text=auto

# Explicitly declare text files we want to always be normalized and converted
# to native line endings on checkout.
*.md            text
*.gitattributes text

# Declare files that will always have CRLF line endings on checkout.
*.ps1    text  eol=crlf
*.psm1   text  eol=crlf
*.psd1   text  eol=crlf
*.psc1   text  eol=crlf
*.ps1xml text  eol=crlf
*.clixml text  eol=crlf
*.xml    text  eol=crlf
*.txt    text  eol=crlf

# Denote all files that are truly binary and should not be mergeable.
*.dll binary
*.exe binary 
```
So looks like my files are forced to have CRLF endings.
Sure if is more convenient for you, you can make all necessary changes by yourself.

**Comment by beatcracker** [comment]
I've pushed a `LF -> CRLF` fix and looks like it worked. I can also add your name here, so you don't have to deal with the files again:
```
- 0.0.7 - Added VS2015 support ([@GDI123](https://github.com/GDI123))
```

**Comment by GDI123** [comment]
Thanks.
I also experienced the security issue when tried to run script in Windows 10 due to the running scripts is prohibited by default. I've add the note to Readme.md how to pass it. Got info from this [link](https://winaero.com/blog/change-powershell-execution-policy-windows-10/).

**Comment by beatcracker** [comment]
Good, but looks like CRLF is messed up again. Let me know when you're done and I'll fix line-endings and merge PR.

**Comment by GDI123** [comment]
Ok, I think I finished changes.

**Comment by beatcracker** [comment]
Awesome, thank you!

**Pull request merged**